ci/build_alpine_container/Makefile

```diff
@@ -0,0 +1,28 @@
+PHONY: all
+
+ENVOY_TAG?="latest"
+
+all: binary container test
+
+binary:
+	rm -rf ./bin
+	mkdir -p ./bin/stripped
+	mkdir -p ./bin/configs
+	docker run -t -i --name build-envoy-alpine-ct -v ${PWD}/../../:/source lyft/envoy-build:latest /bin/bash -c \
+		"set -x && cd /source && /source/ci/do_ci.sh normal && \
```

Comment:
We're about to move away from cmake in the next week or so to Bazel. This CI build flow should be retained, so I don't think it will break anything for you. Just an FYI.